### DISCUSSION TITLE
Add support for Azure Communications Service as an email mechanism

### DISCUSSION THREAD

**Discussion by misalcedo** [nextcloud/all-in-one#3065] [Ideas] [answered]
It would be great to support ACS as a mail provider. Auth0 supports this in their system: https://auth0.com/docs/customize/email/smtp-email-providers/configure-azure-comm-service-as-smtp-email-provider and that is what I have in mind in terms of adding support here.

The idea is to add ACS as a option from the SMTP and Sendmail dropdown that takes a connection string instead of user name and password.

When hosting nextcloud on Azure, I want to avoid running my own SMTP server or relying on an existing mail provider.

**Comment by szaimen** [comment, marked as answer]
Hi, feel free to create a feature request for the here: https://github.com/nextcloud/server/issues
